lambdas/service/app.py

```diff
@@ -1502,11 +1505,15 @@ def generate_manifest(event, context):
     :return: The URL to the generated manifest
     """
     service = ManifestService(StorageService())
-    presigned_url, was_cached = service.get_manifest(format_=ManifestFormat(event['format']),
-                                                     catalog=event['catalog'],
-                                                     filters=event['filters'],
-                                                     object_key=event['object_key'])
-    return {'Location': presigned_url}
+    manifest = service.get_manifest(format_=ManifestFormat(event['format']),
+                                    catalog=event['catalog'],
+                                    filters=event['filters'],
+                                    object_key=event['object_key'])
+    return {
```

Comment:
This should be extracted to a `Manifest.to_json` method. For the step function state, we can abandon the camel case convention and make the JSON keys match the attribute names.